Mathlib/Topology/Category/CompHausLike/Basic.lean

```diff
@@ -140,7 +155,7 @@ instance (X : CompHausLike.{u} P) : T2Space ((forget (CompHausLike P)).obj X) :=
 variable {P}
 
 /-- If `P` imples `P'`, then there is a functor from `CompHausLike P` to `CompHausLike P'`. -/
-@[simps]
+@[simps map]
```

Comment:
I dropped the `obj` from the `@[simps]` lemmas because it looks like this will cause annoying `simpNF` issues down the line, even though `toCompHausLike_obj` doesn't seem to be used anywhere.